ProcessMaker/Listeners/SecurityLogger.php

```diff
@@ -39,10 +39,19 @@ public function handle($event)
             ]);
         } elseif (array_key_exists($class, $this->eventTypes)) {
             $eventType = $this->eventTypes[$class];
+            $meta = $this->getMeta();
+            $data = [
```

Comment:
Considering the Acceptance Criteria, if this information (ip, browser, os) is not required in the modal, no changes are required and the PR can be cancelled. If information is required to be displayed in the modal then I agree to the changes and the PR can be approved.

Reply:
I understand, I will confirm this with Neal

Reply:
You are right the po says: No modal is required for this item, as all information will be provided in the data table.